supervisor/bootstrap.py

```diff
@@ -319,6 +321,25 @@ def filter_data(event, hint):
             }
         )
 
+        # Sanitize event
+        for tag in event.get("tags", []):
+            if "url" in tag:
+                tag[1] = sanitize_url(tag[1])
+
+        if event.get("request"):
+            if event["request"].get("url"):
+                event["request"]["url"] = sanitize_url(event["request"]["url"])
+
+            for header in event["request"].get("headers", []):
+                if header[0] == "Referer":
+                    header[1] = sanitize_url(header[1])
+
+                if header[0] == "X-Hassio-Key":
+                    header[1] = DUMMY_VALUE
+
+                if header[0] in ["Host", "X-Forwarded-Host"]:
+                    event["request"]["headers"].remove(header)
```

Comment:
Let not remove it, we need it for fix ingress, but it into header dummy replacement above

Reply:
These 2 only contains the hostname like "mydomain.com", just displaying "example.com" seems unneeded?

Reply:
Is more to have the full header around and not hunting ghosts